python/ray/llm/_internal/serve/deployments/cloud_utils.py

```diff
@@ -0,0 +1,536 @@
+from typing import List, Optional, Tuple, Union, Dict
```

Comment:
Add a todo to clean up our cloud utils. I don't think we need all of these complexities.